airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -27,6 +27,10 @@ properties:
     type: object
   spec:
     "$ref": "#/definitions/Spec"
+  metadata:
+    type: object
+    description: For internal use only - used by consumers of declarative manifests for storing related metadata
```

Comment:
what does `internal` mean in this context?

Reply:
I could reword it to say "For internal Airbyte use only" ?

Reply:
That would be useful! but yeah just asking a question, not really requesting changes

Reply:
Added "Airbyte" to the description and added a note telling people not to modify this field manually